cmd/istioctl/main.go

```diff
@@ -403,6 +408,66 @@ and destination policies.
 		},
 	}
 
+	configCmd = &cobra.Command{
+		Use:   "context-create --api-server <ip>:<port>",
```

Comment:
nit: this use line is slightly different from the example section below; `--api-server http://<ip>:port` ?